osu.Game.Tests/Visual/SongSelect/TestSceneBeatmapMetadataDisplay.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Utils;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Screens.Play;
+using osuTK;
+
+namespace osu.Game.Tests.Visual.SongSelect
+{
+    [System.ComponentModel.Description("player loader beatmap metadata")]
+    public class TestSceneBeatmapMetadataDisplay : OsuTestScene
+    {
+        private BeatmapMetadataDisplay display;
+
+        [Resolved]
+        private BeatmapManager manager { get; set; }
+
+        private IReadOnlyList<Mod> randomMods => Ruleset.Value.CreateInstance()
+                                                        .GetAllMods()
+                                                        .OrderBy(_ => RNG.Next())
+                                                        .Take(5)
+                                                        .ToList();
```

Comment:
a property should not be returning different, *non-deterministic* results on subsequent accesses. even if it's test code. i would make this a method.